mappings/net/minecraft/world/World.mapping

```diff
@@ -49,10 +49,20 @@ CLASS net/minecraft/class_1937 net/minecraft/world/World
 		ARG 1 mapState
 	METHOD method_17891 getMapState (Ljava/lang/String;)Lnet/minecraft/class_22;
 		ARG 1 id
-	METHOD method_18023 getEntities (Lnet/minecraft/class_1299;Lnet/minecraft/class_238;Ljava/util/function/Predicate;)Ljava/util/List;
+	METHOD method_18023 getEntitiesByType (Lnet/minecraft/class_1299;Lnet/minecraft/class_238;Ljava/util/function/Predicate;)Ljava/util/List;
+		COMMENT Computes a list of entities of the given type within some region.
+		COMMENT
+		COMMENT <strong>Warning:<strong> If {@code null} is passed as the entity type filter, care should be
+		COMMENT taken that the type argument {@code T} is set to {@link Entity}, otherwise heap pollution
```

Comment:
Type arguments are erased at runtime, how would this make a difference? The heap pollution thing at first glance sounds like a load of rubbish, can you clarify precisely under what conditions this can occur?

Reply:
After a short discussion in discord, this could be fixed by clarifying the wording slightly, from "bad casts" to "unchecked casts". What this is really stating is the obvious, that trying to put this in a `List<LivingEntity>` can contain other entities.

Reply:
This is actually getFilteredEntities, isn't it? The predicate is always applied, the type filter only if non-null. Seems the filtering intent is dominant here.